specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -0,0 +1,1398 @@
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ValidateResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources": {
+      "post": {
+        "summary": "Validate whether a resource can be moved.",
+        "description": "Description for Validate whether a resource can be moved.",
+        "operationId": "ValidateMove",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "moveResourceEnvelope",
+            "in": "body",
+            "description": "Object that represents the resource to move.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CsmMoveResourceEnvelope"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppserviceGithubToken": {
+      "description": "Github access token for Appservice CLI github integration.",
+      "type": "object",
+      "properties": {
+        "accessToken": {
+          "description": "Github access token for Appservice CLI github integration",
+          "type": "string"
+        },
+        "scope": {
+          "description": "Scope of the github access token",
+          "type": "string"
+        },
+        "tokenType": {
+          "description": "token type",
+          "type": "string"
+        },
+        "gotToken": {
+          "description": "True if valid github token received, False otherwise",
+          "type": "boolean"
+        },
+        "errorMessage": {
+          "description": "Error message if unable to get token",
+          "type": "string"
+        }
+      }
+    },
+    "AppserviceGithubTokenRequest": {
+      "description": "Appservice Github token request content.",
+      "required": [
+        "code",
+        "state"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
```

Comment:
Is this a secret? Then please set `x-ms-secret` to true.

Reply:
Added